network-config-analyzer/NetworkConfigQuery.py

```diff
@@ -1185,7 +1184,7 @@ def exec(self, only_captured=True):
                     return QueryAnswer(True, self.name2 + ' intersects with ' + self.name1, output_explanation)
 
         return QueryAnswer(False, f'The connections allowed by {self.name1}'
-                                  f' do not intersect the connections allowed by {self.name2}')
+                                  f' do not intersect the connections allowed by {self.name2}', numerical_result=1)
```

Comment:
Why is the numerical result of Intersects query set as 1 when bool result is False?
Would expect 1 for a bool result True..

Reply:
- The numerical result is not used in other return statements in intersects , so it defaults to 0 and "meaningless" in this query's exec method.
but since intersects returns False (as bool result) in two cases : 
1. the configs are not comparable (from calling the "is_identical_topologies()"
2. when config1 do not intersect config2 
and we want to take only the 2nd case as really "forbidden" connection (later in Forbids.)
I used the numerical result as an indicator of the 2nd case.
if using "1" value is not good , I might use another value or
I might add a flag to the IntersectsQuery class to indicate a forbidden connection when calling the IntersectsQuery exec from Forbids.


I don't think adding numerical result = 1 when bool_result = True will help. because then we'll have to change the return value of the "is_identical_topologies()" which is called by other classes too.

Actually I thought that returning numerical_result = 0 with bool_result = True is good , cause exitting with 0 code (res)  means success

Reply:
ok, let's keep this as it is.